libs/ledger-live-common/src/families/cosmos/js-synchronisation.ts

```diff
@@ -14,21 +14,31 @@ import type { OperationType } from "@ledgerhq/types-live";
 import { getMainMessage } from "./helpers";
 import { parseAmountStringToNumber } from "./logic";
 
-const getBlankOperation = (tx, fees, id): CosmosOperation => ({
-  id: "",
-  hash: tx.txhash,
-  type: "" as OperationType,
-  value: new BigNumber(0),
-  fee: fees,
-  blockHash: null,
-  blockHeight: tx.height,
-  senders: [] as string[],
-  recipients: [] as string[],
-  accountId: id,
-  date: new Date(tx.timestamp),
-  extra: {},
-  transactionSequenceNumber: parseInt(tx.tx.auth_info.signer_infos[0].sequence),
-});
+const getBlankOperation = (tx, fees, id): CosmosOperation => {
+  let transactionSequenceNumber: number;
+
+  try {
+    transactionSequenceNumber = parseInt(tx.tx.auth_info.signer_infos[0].sequence);
+  } catch (e) {
+    transactionSequenceNumber = 0;
```

Comment:
Any idea when why `tx.tx.auth_info.signer_infos[0].sequence` is sometines NaN ?

Reply:
This is more to cover the case where auth_info is null

Reply:
Got you, maybe it should be reflected in the type as well that it can be null then

Reply:
I don't know if it can be null as explained in the description